pkgs/top-level/kodi-packages.nix

```diff
@@ -1,7 +1,11 @@
-{ lib, newScope, kodi }:
+{ lib, newScope, kodi, libretro }:
 
 with lib;
 
+let
+  inherit (libretro) snes9x;
+in
```

Comment:
If this is only used once we might as well move it to the package.

Reply:
This is complicated because `libretro.snes9x` here refers to `pkgs.libretro.snes9x`, whereas below we define a `pkgs.kodiPackages.libretro`... so the name shadows and I chose to do it this way. Better idea?

Reply:
I'm going to merge this now but if you have any better ideas we can always circle back. As always, thanks for your reviews. They are much appreciated.